workspace/utilities/lib/util-oembed-attr.xsl

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<xsl:template name="util-oembed-attr">
+	<xsl:param name="driver" select="driver" />
+	<xsl:param name="url" select="url" />
+	<xsl:param name="id" select="@id" />
+	<xsl:param name="title" select="title" />
+
+	<set>
+		<xsl:attribute name="data-oembed-id">
+			<xsl:choose>
+				<xsl:when test="$driver = 'YouTube'">
+					<xsl:value-of select="$url" />
```

Comment:
Add empty value detection? and then add custom attribute node for debugging?

Reply:
Added empty value detection.. Not sure though about what you want for the debugging node.. something like `dev-url="{$url}"` ?

Reply:
yes this would be pretty nice